src/bijectors/simplex.jl

```diff
@@ -136,6 +136,12 @@ function logabsdetjac(b::SimplexBijector, x::AbstractVector{T}) where {T}
 
     return -lp
 end
+
+# Needed to avoid falling back to `with_logabsdet_jacobian` for matrix inputs.
```

Comment:
Which calls `logabsdetjac` again causing a stackoverflow error (had to check the code to realize this is the underlying problem).